cars/v1/x_pack/README.md

```diff
@@ -1,18 +1,29 @@
-This directory contains example configurations for x-pack:
+# This directory contains example configurations for x-pack:
```

Comment:
To me this sounds odd for a heading? How about "Provided configurations"? Also, for other plugins like `repository-gcs` we don't have a heading at all so maybe just drop it?